Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -365,13 +370,20 @@ WarpX::shiftMF (MultiFab& mf, const Geometry& geom, int num_shift, int dir,
                       [=] AMREX_GPU_DEVICE (int i, int j, int k, int n) noexcept
                 {
                       // Compute x,y,z co-ordinates based on index type of mf
+#if (AMREX_SPACEDIM==1)
+                      Real x = 0.0;
+                      Real y = 0.0;
+                      Real fac_z = (1.0 - mf_type[0]) * dx[0]*0.5;
+                      Real z = j*dx[0] + real_box.lo(0) + fac_z;
```

Comment:
Am I correct that this should be
```suggestion
                      Real z = i*dx[0] + real_box.lo(0) + fac_z;
```
?

Reply:
Oh yes! That should be i. Thank you!!